mkdocs.yml

```diff
@@ -1,11 +1,12 @@
-site_name: Pure Storage FlashBlade REST 1.9 Python SDK
+site_name: Pure Storage FlashBlade REST 1.10 Python SDK
 pages:
 - Overview:
-    - Get Started: index.md
+    - Get started: index.md
```

Comment:
This should be `Get Started:`. The capitalization change needs to get checked in upstream so that it stops sneaking through.
I'll manually fix it prior to merging and open an internal Jira ticket for it.

Reply:
Fixed this in the latest commit. 👍